src/components/Posts/ComposeReply.tsx

```diff
@@ -37,6 +38,7 @@ const styles = {
     padddingY: 0,
     justifyContent: "space-between",
   },
+  emojiContainer: { position: "absolute", zIndex: 1, marginTop: 5 },
 };
```

Comment:
adjust zIndex to be anything above Dialog zIndex in `RepliesModal.tsx`

Reply:
Z-index is fine